go.mod

```diff
@@ -11,9 +11,9 @@ require (
 	github.com/magefile/mage v1.15.0
 	github.com/maxbrunsfeld/counterfeiter/v6 v6.6.2
 	github.com/nozzle/throttler v0.0.0-20180817012639-2ea982251481
-	github.com/sigstore/cosign/v2 v2.0.3-0.20230517070756-fc61f43262cb
+	github.com/sigstore/cosign/v2 v2.0.3-0.20230614173616-4a2f67e796b9
```

Comment:
We can either wait for a new release here or just go ahead as-is.

Reply:
we are planning to cut a release soon, maybe today :)